examples/formbot/Makefile

```diff
@@ -15,10 +15,10 @@ train-core:
 	python -m rasa.core.train -s data/stories.md -d domain.yml -o models/dialogue --debug
 
 run-core:
-	python -m rasa.core.run --core models/dialogue --nlu models/nlu/current --verbose --endpoints endpoints.yml
+	python -m rasa.core.run --core models/dialogue --verbose --endpoints endpoints.yml
```

Comment:
this is not identical -- why did you remove the nlu model :D

Reply:
🙈good spot